package.json

```diff
@@ -20,8 +20,8 @@
     "ui-components"
```

Comment:
should line 4 be updated?

Reply:
Can you brief what do you want to say by this? Thanks beforehand @sshivananda

Reply:
That was a very quick reply !!

I think lines 4 and 5 should be
```
"main": "./dist/src/index.js",
"types": "./dist/src/index.d.ts",
```

Reply:
If you test the code without the changes I have made the lint won't have any input to apply upon. That's why it won't work. Therefore, I have added it. You can try it on your end too. @sshivananda

Reply:
What did you mean by
```If you test the code without the changes I have made the lint won't have any input to apply upon. That's why it won't work. Therefore, I have added it. You can try it on your end too.```

Reply:
Nothing, sorry I misunderstood your statement. Will fix this. @sshivananda